src/seedu/addressbook/Main.java

```diff
@@ -87,10 +89,21 @@ private void runCommandLoopUntilExitCommand() {
             CommandResult result = executeCommand(command);
             recordResult(result);
             ui.showResultToUser(result);
-
+            checkEnhancedAddDeleteCommands(command);
+            	
+            
         } while (!ExitCommand.isExit(command));
     }
 
+	private void checkEnhancedAddDeleteCommands(Command command) {
+		if(command instanceof AddCommand || command instanceof DeleteCommand ){
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.